docs/release/release-process.md

```diff
@@ -18,6 +18,15 @@ level sequential overview for how to run the release process.
        tests in the release branch
     1. **tag**: Run the tag pipeline to tag the release branch with an official semver
     1. **build/publish**: Run build pipeline to publish build with official semver
+1. **release template stacks**: Run the release process for each
+   **template stack** that we maintain. Note that the process for
+   template stacks is slightly different from the stack release process.
+    1. **test** Test builds from the release branch, fix any critical
+       bugs that are found
+    1. **version**: Update all version information in the docs, as
```

Comment:
which docs get updated for template stacks? is that mentioned in more detail later on?

Reply:
Can "release stacks" and "release template stacks" merge? perhaps:

"release stacks: Run the release process for each stack and template stack that we maintain"
"version: Update all version information in the docs and integration tests, where appropriate in the release branch"

Reply:
> Can "release stacks" and "release template stacks" merge?

I considered this . . . they **could** merge, but I decided to keep them separate because the way the jobs are structured is a little different, and the way versions are updated is also different between the two. It seemed like it would be more clear if they are two separate sections.